python/ray/tests/test_multinode_failures_2.py

```diff
@@ -1,78 +1,12 @@
 import sys
-import time
 
 import numpy as np
 import pytest
 
 import ray
-import ray._private.ray_constants as ray_constants
 from ray._private.test_utils import get_other_nodes
 
 
-@pytest.mark.skip(reason="No reconstruction for objects placed in plasma yet")
```

Comment:
I tried re-enabling, but its use of `internal.free` causes the test to fail because those objects aren't reconstructable.